### PR TITLE
MTA-1177 snippet - adding notes to viewing analysis reports

### PR DESCRIPTION
**Pull request by emarcusRH**
https://issues.redhat.com/browse/MTA-1177

I created a snippet to be inserted in the new version of the UI Guide chapter 6 that is being handled in 
https://issues.redhat.com/browse/MTA-2072

### CODE REVIEW

File: docs/topics/snippet-reports-filter-sort.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:_content-type: SNIPPET
+
+
+.Procedure
+
+. In *Migration* view, click *Reports*.
+. In the *Current landscape* pane, select *All questionnaires* or a single questionnaire from the drop-down list to see an overview of the risk distribution (High, Medium, Low, Unassessed/Unknown).
+. In the *Identified risks* pane, filter the report results by selecting from the drop-down list: Questionnaire, Section, Question, Answer, or Risk.
```
Comment (anarnold97) [inline docs/topics/snippet-reports-filter-sort.adoc:8]: I so want to be pedantic and say **Identified risks's** but I will be well-behaved

**Review by anarnold97 — APPROVED** [review]
LGTM. Thanks

File: docs/topics/snippet-reports-filter-sort.adoc (hunk 1 above)
Comment (emarcusRH) [reply docs/topics/snippet-reports-filter-sort.adoc:8]: That's the name of the pane   
;-)

**Comment by anarnold97** [comment]
@emarcusRH - please can you have a look at [my PR](https://github.com/windup/windup-documentation/pull/806)?

Where do you want me to put this snippet? Polite answers only :)

Thanks

**Comment by anarnold97** [comment]
> > @emarcusRH - please can you have a look at [my PR](https://github.com/windup/windup-documentation/pull/806)?
> > Where do you want me to put this snippet? Polite answers only :)
> > Thanks
> 
> It probably should replace the procedure under the heading "Reviewing an analysis report" which is currently at line 13 of your draft in the file _docs/topics/mta-web-reviewing-an-analysis-report.adoc_ I inserted a comment in your PR- #806

Fabbo... On it like a car bonnet

**Comment by emarcusRH** [comment]
Closing this PR as it is incorporated in Andy's PR https://github.com/windup/windup-documentation/pull/806  in the file - docs/topics/mta-web-reviewing-an-analysis-report.adoc

**Pull request closed without merging**